src/interfaces.ts

```diff
@@ -28,6 +29,10 @@ export type Mode = '3days' | 'week' | 'day' | 'custom' | 'month'
 
 export type EventCellStyle<T extends ICalendarEventBase> = ViewStyle | ((event: T) => ViewStyle)
 
+export type CalendarCellStyle =
+  | ViewStyle
+  | ((date: dayjs.Dayjs, hourRowIndex?: Number) => ViewStyle)
```

Comment:
The function style prop `CalendarCellStyle` should accept `Date` type as consistency. and `hourRowIndex` should not accept undefined

```suggestion
export type CalendarCellStyle =
  | ViewStyle
  | ((date: Date, hourRowIndex: number) => ViewStyle)
```